src/main/java/com/epam/ta/reportportal/core/log/impl/DeleteLogHandlerImpl.java

```diff
@@ -85,7 +84,8 @@ public OperationCompletionRS deleteLog(Long logId, ReportPortalUser.ProjectDetai
 		Log log = validate(logId, user, projectDetails);
 		try {
 			logRepository.delete(log);
-			cleanUpLogData(log);
+			ofNullable(log.getAttachment()).ifPresent(attachment -> attachmentRepository.moveForDeletion(Collections.singletonList(
+					attachment.getId())));
```

Comment:
May be little update and add appropriate method to repository will be better
if (log.getAttachment())  {
    attachmentRepository.moveForDeletion(log.getAttachment().getId());
}

Reply:
Method replaced with more appropriate